core/converter/pipeline-converter.js

```diff
@@ -45,6 +73,21 @@ exports.PipelineConverter = Converter.specialize({
         value: undefined
     },
 
+   
+
+    _arrayToSet: {
```

Comment:
Collections offers Set.from(anArray) that does that, let's use it?

Reply:
Good catch! At one point I noticed the constructor wasn't working in IE and it didn't occur to me to require collections/set so this was solution I went with. I'll update this.

Reply:
Realized I misread your comment. Collections adds support for `new Set(anArray)`, so I'd prefer to go with that over `Set.from(anArray)` because the constructor is a part of the spec. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set